.github/ISSUE_TEMPLATE/3_feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for oneDPL
+title: ''
+labels: feature_request
+assignees: ''
+
+---
+
+**Summary**
+A clear and concise description of the request. 
+
+**Problem statement**
+Describe the problem you are trying to solve.
+
+**Preferred solution**
+Document your thoughts on what solution may look like.
```

Comment:
Do we want to word this in a way which makes this section more optional?  
I don't think we want to discourage user from providing feature requests just because they don't have a suggestion on how to accomplish it.

Perhaps we can instead say:
"If you have thoughts on what a solution may look like, please document them here"

Reply:
Thanks, Dan. Updated as suggested.